UPGRADING.md

```diff
@@ -0,0 +1,110 @@
+# 2.0.0 Migration Guide
+
+The 1.0 release of the `google-cloud-monitoring-dashboards` client is a significant upgrade based on a [next-gen code generator](https://github.com/googleapis/gapic-generator-python), and includes substantial interface changes. Existing code written for earlier versions of this library will likely require updates to use this version. This document describes the changes that have been made, and what you need to do to update your usage.
+
+If you experience issues or have questions, please file an [issue](https://github.com/googleapis/python-monitoring-dashboards/issues).
+
+## Supported Python Versions
+
+> **WARNING**: Breaking change
+The 2.0.0 release requires Python 3.6+.
+
+## Method Calls
+
+> **WARNING**: Breaking change
+Methods expect request objects. We provide a script that will convert most common use cases.
+* Install the library
+
+```py
+python3 -m pip install google-cloud-monitoring-dashboards
+```
+
+* The scripts `fixup_dashboard_v1_keywords.py` shipped with the library. It expects
+an input directory (with the code to convert) and an empty destination directory.
+
+```sh
+$ fixup_dashboard_v1_keywords.py --input-directory .samples/ --output-directory samples/
+```
+
+**Before:**
+```py
+from google.cloud.monitoring_dashboard import v1
+client = v1.DashboardsServiceClient()
+# TODO: Initialize `parent`:
+parent = ''
+# TODO: Initialize `dashboard`:
+dashboard = {}
+response = client.create_dashboard(parent, dashboard)
+```
+
+
+**After:**
+```py
+from google.cloud.monitoring_dashboard import v1
+client = v1.DashboardsServiceClient()
```

Comment:
There is a namespace change, so the `after` code would be:
```
from google.cloud import monitoring_dashboard_v1
client = monitoring_dashboard_v1.DashboardsServiceClient()
```

I would recommend add a new section above explaining there is a namespace change, and give the `before` and `after` code explaining how to create a client. In this section, only keep the `response = client.create_dashboard(...)` part. The script itself only modifies the method code, it doesn't fix the namespace.

Reply:
Sounds reasonable to me, I removed the creating client part here and added a section above to explain the breaking change in the sense of creating new client. Please LMK if it looks good to you, thanks.

Reply:
It looks good to me, thanks!